### PR TITLE
Added routes for NetworkDeviceFunctions and NetworkPorts

### PR DESCRIPTION
**Pull request by amar-shalgar**
This PR is part of #239

### CODE REVIEW

File: svc-api/router/router.go [hunk 1]
```diff
@@ -315,9 +315,18 @@ func Router() *iris.Application {
 	chassis.Get("/{id}", cha.GetChassis)
 	chassis.Get("/{id}/NetworkAdapters", cha.GetChassisResource)
 	chassis.Get("/{id}/NetworkAdapters/{rid}", cha.GetChassisResource)
+	chassis.Get("/{id}/NetworkAdapters/{rid}/NetworkDeviceFunctions", cha.GetChassisResource)
+    chassis.Get("/{id}/NetworkAdapters/{rid}/NetworkPorts", cha.GetChassisResource)
```
Comment (a-ajith) [inline svc-api/router/router.go]: please apply go tools

**Review by a-ajith — CHANGES_REQUESTED** [review]

File: svc-api/router/router.go (hunk 1 above)
Comment (amar-shalgar) [reply svc-api/router/router.go]: Addressed review comments

**Review by a-ajith — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]
IMHO the same routes need to be added to GRF plugin please update the same

**Review by irappa-pattar — APPROVED** [review]

**Review by bharath-b-hpe — APPROVED** [review]
Please run go tools.

**Review by Bharath-KKB — APPROVED** [review]

**Pull request merged**